packages/tendermint-rpc/src/tendermint33/adaptor/index.ts

```diff
@@ -1,10 +1,12 @@
 /* eslint-disable @typescript-eslint/naming-convention */
-import { Adaptor } from "../../adaptor";
-import { hashBlock, hashTx } from "../../hasher";
+import { hashBlock, hashTx } from "../hasher";
 import { Params } from "./requests";
 import { Responses } from "./responses";
+import { Adaptor } from "./types";
 
-export const v0_34: Adaptor = {
+export { Decoder, Encoder, Params, Responses } from "./types";
+
+export const adaptor33: Adaptor = {
```

Comment:
The renaming should allow you to drop the `/* eslint-disable @typescript-eslint/naming-convention */`